src/main/java/com/notably/view/MainWindow.java

```diff
@@ -142,6 +145,18 @@ private void setWindowDefaultSize(GuiSettings guiSettings) {
         }
     }
 
+    private HelpWindow initializeHelpWindow(Model model) {
+        model.helpOpenProperty().addListener((Observable observable) -> {
+            if (model.isHelpOpen()) {
+                handleHelp();
+            }
+            model.setHelpOpen(false);
+        });
+        HelpWindow helpWindow = new HelpWindow();
+        requireNonNull(helpWindow);
+        return helpWindow;
+    }
+
```

Comment:
Actually, it feels rather unintuitive for me to have the listener set up before the `helpWindow` property is initialized. There's a part of me that is worried about what will happen if the listener is fired before `helpWindow` is initialized (this might cause an NPE)

I was thinking you could rearrange it a bit as such? (I made the `initializeHelpWindow` function return void. You won't need `helpWindow = initializeHelpWindow(model)` anymore if you do this)
```java
private void initializeHelpWindow(Model model) {
    helpWindow = new HelpWindow();

    model.helpOpenProperty().addListener((Observable observable) -> {
        if (model.isHelpOpen()) {
            handleHelp();
        }
        model.setHelpOpen(false);
    });
}
```

Reply:
Thanks for the feedback.